doc/nrf/ug_bt_qualification.rst

```diff
@@ -0,0 +1,109 @@
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Every product using the Bluetooth® LE technology must be certified (qualified) by Bluetooth Special Interest Group (SIG).
+This condition must be met to perform the following actions:
+
+* Demonstrate that your products satisfy the requirements of the Bluetooth license agreements and specifications.
+* Use Bluetooth trademark and logos.
+* Sell products that use Bluetooth LE technology.
+
+As a result, your products will be listed and can be promoted on the official Bluetooth SIG page.
+
+
+Pre-qualified design
+********************
+
+By using pre-qualified designs, you can omit the testing phase during your product's qualification.
+
+The following product types are used and qualified by Nordic Semiconductor:
+
+* End product
+* Controller Subsystem product
+* Host Subsystem product
+* Profile Subsystem product
+* Component product
+
+All versions of Controller Subsystems (:ref:`ug_ble_controller_softdevice` versions), Host Subsystems and Components are created and qualified by Nordic Semiconductor by default.
+Some Profile Subsystems (e.g., :ref:`ug_bt_mesh` versions) are qualified as well.
+If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_ yet, run the tests through the `Profile Tuning Suite (PTS)`_ tool and qualify the needed product by yourself.
+
+
+Qualification process
+*********************
+
+The product qualification cannot be inherited, hence you must complete this process by yourself.
+You can either perform a full qualification process for a new product, or edit an existing entry for a product that has already been qualified.
+
+New entry creation
+==================
+
+The only way to qualify your product is to complete the `Bluetooth SIG's Qualification Process`_ under your member company's account.
+To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.
+
+Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following paths:
+
+* `Qualification process with no required testing`_ - If your complete product uses only pre-qualified End products or Subsystem products.
+* `Qualification process with required testing`_ - If your complete product uses a new Bluetooth design or pre-qualified Component products.
+
+.. note::
+   A *design* is a specific configuration of hardware or software implementation of adopted Bluetooth specifications.
+
+
+Existing entry update
+=====================
+
+Update an existing qualification entry through `Launch Studio`_ in the following cases:
+
+* You changed an existing product's design significantly so that the testing is required.
+* Your product can be added as a new model to one of your existing qualifications.
+* You switched to another SDK version for an existing product.
+
+If you want to switch to another SDK version for an existing product, perform the following steps:
```

Comment:
```suggestion
* You switched to another SDK version for an existing product.

Switching to another SDK version
--------------------------------

If you want to switch to another SDK version for an existing product, complete the following steps:
```
"complete" has most occurrences in the SDK docs in this context, so let's stay consistent.